### PR TITLE
Add 2 P2P provider bootstrap nodes

### PR DESCRIPTION
**Pull request by antonr-p2p**

### CODE REVIEW

File: config/_peers/mainnet [hunk 1]
```diff
@@ -3,3 +3,5 @@
 /dns4/keep-validator-0.prod-eks-eu-west-1.staked.cloud/tcp/3919/ipfs/16Uiu2HAm6Fs6Fn71n7PqRmpHMbfMkZUCGYhW5RL81MSMg57AANkZ
 /dns4/keep-validator-1.prod-eks-ap-northeast-2.staked.cloud/tcp/3919/ipfs/16Uiu2HAm5UzZb1TTYBjb2959h4z4VHzjt585SQqZnJPBrDnJuob7
 /dns4/keep-validator-2.prod-eks-eu-north-1.staked.cloud/tcp/3919/ipfs/16Uiu2HAmJvbYNhzY6a8kiG2zzrqXGnYWax7CQTbiMHoAvY4qLvg7
+/dns4/bootstrap-b1.threshold.p2p.org/tcp/3919/ipfs/16Uiu2HAmG7kjmp4D3C9jkvA8s59tfJzPfJSVqQJkhG6tEvVhJW68
+/dns4/bootstrap-b2.threshold.p2p.org/tcp/3919/ipfs/16Uiu2HAmSuXryEFqeDaMmRZ7KeUdQmHCt3PiWVNpe37q2xR3VT5n
```
Comment (pdyraga) [inline config/_peers/mainnet:7]: Hey @antonr-p2p, I think there is something off with `bootstrap-b2.threshold.p2p.org`. Can you double-check, please?

```
❯ telnet bootstrap-b1.threshold.p2p.org 3919
Trying 34.140.78.107...
Connected to bootstrap-b1.threshold.p2p.org.
Escape character is '^]'.
/multistream/1.0.0
^CConnection closed by foreign host.

❯ telnet bootstrap-b2.threshold.p2p.org 3919
Trying 35.228.156.5...
telnet: connect to address 35.228.156.5: Connection refused
telnet: Unable to connect to remote host
```

Comment (antonr-p2p) [reply config/_peers/mainnet:7]: @pdyraga fixed

**Review by pdyraga — APPROVED** [review]

**Comment by antonr-p2p** [comment]
> Oops, looks like the commit is unsigned. Can you please set up GPG commit signing and rebase @antonr-p2p?
> 
> Here are the instructions: https://docs.github.com/en/authentication/managing-commit-signature-verification

I don't want to setup signing, please, create a new PR from your account with these just 2 lines.

**Comment by pdyraga** [comment]
Closing this PR in favor of https://github.com/keep-network/keep-core/pull/3555.

**Pull request closed without merging**